etc/templates/config/windows/2003/profile.template

```diff
@@ -0,0 +1 @@
+<config-profile>windows, windows2003</config-profile>
```

Comment:
Add more ways to call a Windows Server 2003, for example, `windows-server`, `windows-server-2003`. Same for the rest of Windows Server versions.